build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+VERSION=$(git describe --tags --always --dirty 2>/dev/null || echo "dev")
```

Comment:
could we get a version of this script that automatically pulls down the go compiler as well for use day of?

Reply:
Yeah the script is very bare-bones, just wanted a starting point. Think that's a good idea for another PR.